.eslintrc.js

```diff
@@ -22,6 +22,7 @@ module.exports = {
       'error',
       {
         '*': 4,
+        'curated-card': 7,
```

Comment:
@thomsebastin  this will be 'curated-card' as we have more than 5 authoring fields under 'curated-cards' block